python/ray/serve/_private/deployment_scheduler.py

```diff
@@ -192,13 +200,36 @@ def _schedule_spread_deployment(self, deployment_name: str) -> None:
                 pending_replica_name
             ]
 
+            placement_group = None
+            if replica_scheduling_request.placement_group_bundles is not None:
+                strategy = (
+                    replica_scheduling_request.placement_group_strategy
+                    if replica_scheduling_request.placement_group_strategy
+                    else "PACK"
```

Comment:
Do we set `PACK` by default because we expect the user to use placement groups to schedule replicas and related Ray actors/tasks together? So we want to maintain locality between the replicas and their related actors/tasks?

Reply:
Based on the unit tests, it looks like co-scheduling Ray actors/tasks with Serve replicas isn't the main purpose of this change. I'm not sure if my question is relevant. Feel free to resolve if it's not.

Reply:
`"PACK"` is just the default strategy in Ray so using the same default

Reply:
No, the `"PACK"` applies to the placement group, not among the placement groups